README.md

```
@@ -1 +1,5 @@
-# sprocket
+# Sprocket
+
+gdscript linting and formatting rewrite/port of gdtoolkit-py
+
+[gdtoolkit](https://github.com/Scony/godot-gdscript-toolkit] provides tools for linting and formatting GDScript code, ensuring consistent code style and quality. However I was frustrated with the amount of time it'd take to format and lint files (often several seconds) so i decided to rewrite it all in go
```

markdownlint-cli2 (0.17.2)

Bare URL used
null

(MD034, no-bare-urls)